x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/index.tsx

```diff
@@ -0,0 +1,259 @@
+      if (ip != null) {
+        return (
+          <DraggableWrapper
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id: escapeDataProviderId(`networkTopNFlow-table-${networkTopNFlowType}-ip-${ip}`),
+              name: ip,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: attr,
+                value: ip,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                ip
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DOMAIN,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) => {
+      const attr =
+        networkTopNFlowType === NetworkTopNFlowType.source ? 'source.domain' : 'destination.domain';
+      const domain: string | null = get(attr, node);
+      if (domain != null) {
+        return (
+          <DraggableWrapper
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id: escapeDataProviderId(
+                `networkTopNFlow-table-${networkTopNFlowType}-domain-${domain}`
+              ),
+              name: domain,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: attr,
+                value: domain,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                domain
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.BYTES,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.bytes', node)
+        ? numeral(node.network!.bytes).format('0.000b')
+        : getEmptyTagValue(),
+  },
+  {
+    name: i18n.PACKETS,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.packets', node)
+        ? numeral(node.network!.packets).format('0,000.00')
+        : getEmptyTagValue(),
+  },
+  {
+    name: i18n.DURATION,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('event.duration', node)
+        ? moment.duration(node.event!.duration! / 1000000, 'ms').humanize()
```

Comment:
do you need to use the `i8n tooling` of relative here for the relative date time text?

Reply:
oh yes you are correct

Reply:
https://github.com/elastic/ingest-dev/issues/271